temba/channels/models.py

```diff
@@ -1718,7 +1718,13 @@ def send_vumi_message(cls, channel, msg, text):
         headers = dict(TEMBA_HEADERS)
         headers['content-type'] = 'application/json'
 
-        url = 'https://go.vumi.org/api/v1/go/http_api_nostream/%s/messages.json' % channel.config['conversation_key']
+        # for backwards compatibility
+        if not channel.config.get('api_url'):
+            api_url_base = 'https://go.vumi.org/api/v1/go/http_api_nostream'
+        else:
+            api_url_base = channel.config['api_url']
+
+        url = '%s/%s/messages.json' % (api_url_base, channel.config['conversation_key'])
```

Comment:
Should build the path using URL lib.. ie:

```python
urlparse.urljoin(api_url_base, "%s/messages.json" % channel.config['conversation_key'])
```
That'll take care of any issues with repeating /'s